src/main/java/pl/javastart/bootcamp/domain/admin/AdminUserController.java

```diff
@@ -18,10 +23,29 @@ public AdminUserController(UserService userService) {
 
     @GetMapping("/admin/uzytkownicy")
     public String users(Model model) {
+        List<User> users = userService.findAll();
+        Map<Long, Boolean> isAdminMap = users.stream()
+                .collect(Collectors.toMap(
+                        User::getId,
+                        userService::isUserAdmin
+                ));
+
         model.addAttribute("users", userService.findAll());
+        model.addAttribute("isAdminMap", isAdminMap);
         return "admin/users";
```

Comment:
Tutaj 2x wywołujesz `findAll()`.
Dodatkowo, zamiast tworzyć mapę, to lepiej dodać pole `isAdmin` do obiektu. Stwórz proszę dedykowaną klasę UserDto z polami potrzebnymi do wyświetlania na liście.